Model/Order/StatusUpdater.php

```diff
@@ -126,6 +181,12 @@ public function success(string $orderIncrementId, int $transactionId)
 
             $this->orderRepository->save($order);
             $this->orderCommentSender->send($order, true, $comment);
+
+
+            /** Create Invoice if order was virtual */
+            if ($order->getIsVirtual()) {
```

Comment:
There should be a config checking here to check if admin approved the invoicing in the store. Many of shop owners don't want to create invoice inside Magento for different reasons.